lib/content-services/src/lib/content-metadata/services/content-metadata.service.ts

```diff
@@ -87,13 +86,25 @@ export class ContentMetadataService {
     }
 
     setTitleToNameIfNotSet(propertyGroups: OrganisedPropertyGroup[]): OrganisedPropertyGroup[] {
+        const propertyGroupsTitles = {};
```

Comment:
Instead of a JSON object containing data like 
```
propertyGroupTitles = {
   aspectTitle: 'aspectTitle',
   someOtherTitle: 'someOtherTitle'
}
```
we can instead have an array based collection, like
```
propertyGroupTitles = [ 'aspectTitle', 'someOtherTitle' ]
```
and then use `propertyGroupTitles.includes(title)` in the if check. That way, the entire logic would be much more readable on first glance

Reply:
@swapnil-verma-gl thanks for pointing that out. I've applied the suggested change and added a new unit test to cover it.